security/advancedtls/advancedtls_test.go

```diff
@@ -381,14 +381,14 @@ func (s) TestClientServerHandshake(t *testing.T) {
 		return &GetRootCAsResults{TrustCerts: cs.ServerTrust3}, nil
 	}
 
-	makeStaticCRLProvider := func(crlPath string) *RevocationConfig {
+	makeStaticCRLProvider := func(crlPath string, allowUndetermined bool) *RevocationConfig {
```

Comment:
Can we rename this to something like `MakeRevocationConfigWithStaticCrlProvider`?

Reply:
Since it's private for the test function, I changed it to a shorter `makeStaticCRLRevocationConfig`